torchao/dtypes/fpx/fpx.py

```diff
@@ -36,8 +51,40 @@ def _bit_interleave(x: Tensor, n_bits: int, undo: bool = False) -> Tensor:
 
     if not undo:
         bit_order = {
-            1: [1, 5, 9, 13, 17, 21, 25, 29, 3, 7, 11, 15, 19, 23, 27, 31,
-                0, 4, 8, 12, 16, 20, 24, 28, 2, 6, 10, 14, 18, 22, 26, 30],
+            1: [
+                1,
```

Comment:
lol this is a fun result of this lint...

Reply:
This is auto-formatted in ruff. If I change the format, the CI/CD test for ruff format --check fails.

Reply:
does ruff support some kind of skipping a block of code? I know black has `# fmt: off/on`. This is one of the cases where auto-format tools make things worse 😄